### PR TITLE
[DS-360]: [Pyxis Documentation] Add Autocomplete code documentation on Zeroheight

### PR DESCRIPTION
**Pull request by brianvanburken**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/DS-360

## 📝 Docs
The documentation of the Autocomplete component has been updated.

### CODE REVIEW

File: packages/pyxis-react/src/stories/components/Autocomplete/Overview.tsx [hunk 1]
```diff
@@ -181,7 +179,16 @@ const accessibilityDescription = (
 
 const classDescription = (
   <p>
-    Autocomplete is based upon this list of CSS classes.
+    The list of Texts CSS classes is available&nbsp;
```
Comment (lauratrapani) [inline packages/pyxis-react/src/stories/components/Autocomplete/Overview.tsx:182]:
```suggestion
    The list of Autocomplete CSS classes is available&nbsp;
```

**Comment by lauratrapani** [comment]
Also, the label and description of the PR are missed

File: packages/pyxis-react/src/stories/components/Autocomplete/Overview.tsx [hunk 2]
```diff
@@ -181,7 +179,16 @@ const accessibilityDescription = (
...
+    <a
+      className="link"
+      href="https://prima.design/3794e337c/p/26b9e3-autocomplete"
```
Comment (emanuelemacri) [inline packages/pyxis-react/src/stories/components/Autocomplete/Overview.tsx:185]: we generally use the direct link to the 'code' tab
```suggestion
      href="https://prima.design/3794e337c/p/26b9e3-autocomplete/b/54df21"
```

**Review by emanuelemacri — CHANGES_REQUESTED** [review]
in addition to laura's comments: before merging PR you have to finish the documentation on zeroheight. ;)

**Review by lauratrapani — APPROVED** [review]

**Review by emanuelemacri — APPROVED** [review]
LGTM

**Pull request merged**